### PR TITLE
Display time remaining indicators for each of the current limited multipliers

### PR DESCRIPTION
**Pull request by joshuaclayton**
What?
=====

This displays the current limited multipliers, as well as pie charts
displaying the amount of remaining time, so players can understand how
much longer a multiplier will be active.

### CODE REVIEW

File: src/elm/Data/Expirable.elm [hunk 1]
```diff
@@ -15,13 +16,17 @@ type SecondsRemaining
...
+type SecondsTotal
+    = SecondsTotal Time.Time
+
+
 type Expirable a
-    = Expirable a SecondsRemaining
+    = Expirable a SecondsRemaining SecondsTotal
 
 
 expiresIn : SecondsRemaining -> a -> Expirable a
```
Comment (johnschoeman) [inline src/elm/Data/Expirable.elm:27]: What do you think about `buildExpirableFromTotalSeconds`?

**Review by johnschoeman — APPROVED** [review]
LGTM!
one minor thing.

File: src/elm/Data/Expirable.elm (hunk 1 above)
Comment (joshuaclayton) [reply src/elm/Data/Expirable.elm:27]: Hmm - I'm not sure I'm onboard with that amount of verbosity, but I'm interested in hearing why you feel that better clarifies intent over `expiresIn`!

Comment (johnschoeman) [reply src/elm/Data/Expirable.elm:27]: I was looking for a name that expresses that this function builds something.  When first reading it, i expected the behavior to be such that it takes an Expirable and returns how many seconds are remaining.

Comment (joshuaclayton) [reply src/elm/Data/Expirable.elm:27]: Ah, huzzah - how about `build`, which is a pattern used throughout the codebase? This can happen in a separate PR.

**Pull request merged**